tests/envs/test_envs.py

```diff
@@ -80,13 +78,7 @@ def test_env(self, env):
         a = act_space.sample()
         assert act_space.contains(a)
         res = env.step(a)
-        _, _, _, _ = env.step(a)
         assert ob_space.contains(res[0])  # res[0] --> observation
         assert np.isscalar(res[1])  # res[1] --> reward
         env.render()
-        if all(not isinstance(env, T) for T in [GridWorldEnv, Box2DEnv]):
```

Comment:
please leave this here. it verifies that off-screen rendering is working.

Reply:
Actually this breaks the test. There are multiple assertion errors, i.e. `img is not None`

Reply:
then the code is broken, not the test.